yarn.lock

```diff
@@ -6900,6 +7324,104 @@ jszip@^3.5.0:
     readable-stream "~2.3.6"
     setimmediate "^1.0.5"
 
+karma-browserstack-launcher@^1.4.0:
```

Comment:
These are devDependencies in terriajs, not sure how they got into TerriaMap's yarn.lock.

Reply:
This happens during dev stage when doing `yarn install` with terriajs checked out inside `packages/` workspace. Goes away when you install normally without using workspace.